dataactbroker/handlers/fileHandler.py

```diff
@@ -1113,6 +1113,18 @@ def restart_validation(submission):
 
         return JsonResponse.create(StatusCode.OK, {"message": "Success"})
 
+    def fail_validation(self, upload_id):
+        # update all validation jobs to "ready"
+        sess = GlobalDB.db().session
+
+        jobs = sess.query(Job).filter(Job.submission_id == upload_id).all()
```

Comment:
I clarified with Minahm. Upload ID = submission ID so this isn't applicable. Clearer naming was requested.